config/script.go

```diff
@@ -43,6 +43,8 @@ type ScriptCmdOptions struct {
 	CloudTerraformPlanFile string
 	CloudTofuPlanFile      string
 	UseTerragrunt          bool
+	EnableSharing          bool
+	MockOnFail             bool
```

Comment:
*nitpick* Should we name it like the flag name because it may be misleading mixed with other concepts? This applies to script configuration too
```suggestion
	MockSharingOnFail             bool
```

Reply:
It makes sense! 
I'm waiting for @mariux feedback if the `mock_sharing_on_fail` name is okay, so once this is decided I will change.